anomaly_detector/config.py

```diff
@@ -51,9 +51,9 @@ class Configuration():
     MODEL_PATH = ""
     W2V_MODEL_PATH_CALLABLE = join_w2v_model_path
     W2V_MODEL_PATH = ""
-    MODEL_STORE= ""
+    MODEL_STORE= os.getenv("LAD_MODEL_STORE","")
```

Comment:
You don't need to do this - just define the name of the config option (`MODEL_STORE`) and the `Configuration.load_from_env` will automatically match the name to env variable and set it.

Reply:
@vpavlin what about `LOCAL_DATA_FILE` and `LOCAL_RESULTS_FILE` ? Those seem to have been set with env vars.

Reply:
that's right. You should make all setting consistent. 
Looking at the code below, you'll want all environment variables prefixed with `LAD` in `.env.example`. 
For testing in travis or locally you might want to use another prefix as `LAD_TEST` - but not sure. Seems [here](https://github.com/AICoE/log-anomaly-detector/blob/master/tests/test_validation_data.py#L5) you're using `LAD` but in `.env.example` you've been using `LADT`

Reply:
The original file had that but I think its a typo. I'll remove that extra 'T'

Reply:
Some of the env vars that are in this file are different from those provided in:
https://github.com/AICoE/log-anomaly-detector/blob/master/openshift/log-anomaly-detector.app.yaml#L87-L126

```
- name: LAD_STORAGE_BACKEND
            - name: LAD_MODEL_DIR
            - name: LAD_MODEL_FILE
            - name: LAD_W2V_MODEL_FILE
            - name: LAD_TRAIN_TIME_SPAN
            - name: LAD_TRAIN_MAX_ENTRIES
            - name: LAD_TRAIN_ITERATIONS
            - name: LAD_TRAIN_UPDATE_MODEL
            - name: LAD_PARALLELISM
            - name: LAD_INFER_TIME_SPAN
            - name: LAD_INFER_MAX_ENTRIES
            - name: LAD_INFER_LOOPS
            - name: LAD_INFER_ANOMALY_THRESHOLD
            - name: LAD_ES_ENDPOINT
            - name: LAD_ES_INPUT_INDEX
            - name: LAD_ES_TARGET_INDEX
            - name: LAD_ES_QUERY
            - name: LAD_ES_CERT_DIR
```

Reply:
I think we should get rid of .env file and just add proper documentation on what the fields mean. I could add a very simple .env file with basic fields to train locally

Reply:
Lets move this to another PR. I'll open an issue for this now https://github.com/AICoE/log-anomaly-detector/issues/69